pkg/spanconfig/spanconfig.go

```diff
@@ -66,3 +62,9 @@ type Update struct {
 	// Deleted is true if the span config entry has been deleted.
 	Deleted bool
 }
+
+// SQLWatcher is responsible for reacting to sql changes that imply span
+// configuration updates.
+type SQLWatcher interface {
```

Comment:
Move this declaration (also for KVWatcher) above ReconciliationDependencies. Re-use the comment in the sketch API from earlier.